src/fixtures/configs/snowflake-transformation.json

```diff
@@ -7,7 +7,7 @@
         {
           "codes": [
             {
-              "name": "Code 1",
+              "name": "Code X",
```

Comment:
Zjednotene testovacie data, ... `blocks` cislovane cislom a `codes` znakom ... bolo to rozne medzi SNFK a PY testovacimi configmi, a mylilo ma to.